bower.json

```diff
@@ -5,7 +5,7 @@
     "bootstrap": "^3.3.6",
     "chosen": "~1.4.2",
     "datatables": "#1.10.2",
-    "dcjs": "~2.0.0",
+    "dcjs": "2.0.0-beta.31",
```

Comment:
When you delete the bower_components folder and run the 'bower install' command, it would download 2.0.0-beta.32 version. With this version we would be running into issue in function [filterLabels()](https://github.com/cBioPortal/iViz/blob/multi-cohort/app/scripts/views/components/pieChart/pieChart.js#L266) because the svg structure generated is different compared to the one generated to dcjs 2.0.0-beta.31 version.